hack/chartverifier_test.go

```diff
@@ -0,0 +1,33 @@
+package hack
```

Comment:
how about that we keep `hack` folder free from go files and move tests under `/test` folder in the project root?

Reply:
Moved in f646ba3.